code/style.css

```diff
@@ -1,4 +1,213 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
+  margin: 0;
+  background: white
 }
+
+
+header {
+  display: block;
+  height: 200px;
+  background-color: #ffffff;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+}                                 
+
+
+.logo-img {
+  width: 250px;
+  margin: 0 0 0 2%;                    
+}
+
+
+nav {
+  align-self: flex-end;                    
+  margin: 14px;
+}
+
+
+.topnav {
+  background-color: #ffff;
+  overflow: hidden;
+  font-family: 'Bebas Neue', cursive;
+  
+}
+
+
+.topnav a {
+  float: left;
+  color: #c0c0c0;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 25px;
+}
+
+
+.topnav a:hover {
+  background-color: #c59000;
+  color: #393e46;
+}  
+
+
+article {
+  background: #FFF8F0; 
+  padding: 20px;
+}
+
+
+.big-news {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+
+.big-news-img {
+  width: 100%;
+  margin: 10px;
+  border-radius: 43px;
+  background: #e0e0e0;
+  box-shadow:  20px 20px 31px #9b9b9b,
+               -20px -20px 31px #ffffff;
+}
+
+
+h1 {   
+  color: #A68D65;
+  margin: 10px;
+  font-family: 'Bebas Neue', cursive;
+  font-size: 88px;
+}
+
+
+.small-news {
+  background-color: white;
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  object-fit: none;              
+  align-items: center;
+}
+
+
+.news-box {
```

Comment:
I left another comment in the end (row 182). A suggestion is to set a max-height on .news-box {max-height: XXXpx; width: 350px; object-fit: cover;} and add another width for your media query on bigger screens, then the images will looks reallly good on all screens :)